### PR TITLE
Add tiles as a countable value

### PR DESCRIPTION
**Pull request by PLynx01**
After somewhat long time of inactivity, I decided to come back with something new. It's the possibility to use tiles as a countable.

It's very likely that it contains bugs, so please point them out and explain me. That will enable me to avoid them in my future contributions.

If you have some remarks, please share it concisely, whenever possible.

### CODE REVIEW

File: core/src/com/unciv/models/ruleset/unique/Countables.kt [hunk 1]
```diff
@@ -33,6 +33,9 @@ object Countables {
             return civInfo.cities.sumOf { it.cityConstructions.getBuiltBuildings()
                 .count { it.matchesFilter(placeholderParameters[0]) } }
 
+        if (countable.equalsPlaceholderText("[] Tiles"))
+            return gameInfo.tileMap.tileList.count { it.matchesFilter(placeholderParameters[0]) }
```
Comment (yairm210) [inline core/src/com/unciv/models/ruleset/unique/Countables.kt:37]: Tiles in the *entire map*?! That's both very unintuitive, and I can't really see a use case for this
All the other filters are for civ-wide counts, not map-wide

Comment (PLynx01) [reply core/src/com/unciv/models/ruleset/unique/Countables.kt:37]: So it should be only civ-wide?

What should we use instead for obtaining map area, height, width or amount of land and water tiles?

Comment (yairm210) [reply core/src/com/unciv/models/ruleset/unique/Countables.kt:37]: Who even wants that?

Comment (PLynx01) [reply core/src/com/unciv/models/ruleset/unique/Countables.kt:37]: Please explain me your advices in more straightforward and unambiguous way.

So I understand that it tile-counting should be civ-wide, not map-wide. Am I right?

**Pull request closed without merging**